tests/unit/test_backup.py

```diff
@@ -494,7 +494,7 @@ def test_exists_grpc_error(self):
         from google.api_core.exceptions import Unknown
```

Comment:
These methods all need to be updated the same way:
- `test_create_*`,
- `test_get`
- `test_reload`
- `test_delete_*`
-  `test_update_*`
- `test_restore_*`
- `test_get_iam_policy`
- `test_set_iam_policy`
- `test_test_iam_permissions`